src/gazebo_motor_model.cpp

```diff
@@ -33,7 +33,8 @@ void GazeboMotorModel::InitializeParams() {}
 
 void GazeboMotorModel::Publish() {
   turning_velocity_msg_.set_data(joint_->GetVelocity(0));
-  motor_velocity_pub_->Publish(turning_velocity_msg_);
+  // FIXME: Commented out to prevent warnings about queue limit reached.
+  // motor_velocity_pub_->Publish(turning_velocity_msg_);
```

Comment:
I don't actually think we need this at all. Better just remove it.

Reply:
Ok, I have no idea...

Reply:
I think this is used by the motor_failure_plugin. It was on the todo list of mine to remove the ROS dependency...but also fine with removing it

Reply:
Ok, I guess let's have it commented out for now.